src/Hue.cpp

```diff
@@ -445,7 +489,7 @@ void Hue::refreshState()
         return;
     }
     nlohmann::json answer = commands.GETRequest("", nlohmann::json::object(), CURRENT_FILE_INFO);
-    if (answer.is_object() && answer.count("lights"))
+    if (answer.is_object() && (answer.count("lights") || answer.count("sensors")))
```

Comment:
Wouldn't an explicit `bool Hue::sensorExists(int id)` be better? At least to reduce confusion?